src/generators.rs

```diff
@@ -147,6 +147,24 @@ pub fn make_sized_tuple_sigs_map(input_sizes: Vec<u16>) -> HashMap<u16, TupleTyp
     ret_map
 }
 
+fn helper_make_clarity_type_for_sized_type_sig(input_size: u16) -> String {
+    match input_size {
+        1 => "uint".to_string(),
+        2 => "(optional bool)".to_string(),
+        n => {
+            let mult = n / 8;
+            // (tuple (key-name-0 key-type-0) (key-name-1 key-type-1) ...)
+            let mut key_pairs = String::new();
+            for i in 0..mult {
+                let id_name = if i < 10 { "id--" } else { "id-" };
```

Comment:
What's the purpose of varying the amount of dashes?

Reply:
Discussed offline: updating to work with larger input sizes